web/src/environments/environment.ts

```diff
@@ -7,12 +7,12 @@ export const environment = {
   version: 'x.x.x',
   // Add dev firebase credentials
   firebase: {
-    apiKey: '',
-    authDomain: '',
-    databaseURL: '',
-    projectId: '',
-    storageBucket: '',
-    messagingSenderId: ''
+    // apiKey: '',
+    // authDomain: '',
+    // databaseURL: '',
+    // projectId: '',
+    // storageBucket: '',
+    // messagingSenderId: ''
```

Comment:
Why did these get commented out?

Reply:
I commented out as I was using my firebase credentials to run in local. It was easy for me to comment and use my credentials. Will uncomment it and will commit again.